schemas/com.snowplowanalytics.snowplow/media_player_ad/jsonschema/1-0-0

```diff
@@ -0,0 +1,63 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a context entity with information about the currently played ad",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow",
+    "name": "media_player_ad",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "description": "Friendly name of the ad.",
+      "type": [
+        "null",
+        "string"
+      ],
+      "maxLength": 4096
+    },
+    "adId": {
+      "type": "string",
+      "format": "uuid",
```

Comment:
That's a great point, UUID was too restrictive as the ID will be passed from users. Thinking about this, I also changed the `mediaSessionId` and `breakId` in the session and ad-break schemas, because those might also be passed by users and with different formats.